packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -330,4 +330,22 @@ describe('Date Picker', () => {
     Terra.should.matchScreenshot();
     Terra.should.beAccessible({ rules: ignoredA11y });
   });
+
+  describe('Required Date Picker', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-required'));
+
+    it('creates a hidden input and empty value attr. when no date is entered', () => {
```

Comment:
Don't worry about checking for attributes inside the wdio tests. Those are best left to the jests tests.

Reply:
Oh I see. I removed the attribute checks: https://github.com/cerner/terra-framework/pull/504/commits/49decfbc4d5c6a3500eff04684e94fbe0c23eb30